testing/scenario_app/ios/Scenarios/ScenariosTests/AppLifecycleTests.m

```diff
@@ -140,5 +140,13 @@ - (void)testLifecycleChannel {
   XCTAssertEqualObjects(
       lifecycleEvents, expectedStates,
       @"AppLifecycleState transitions while presenting a second time not as expected");
+
+  // Dismantle.
+  [engine.lifecycleChannel setMessageHandler:nil];
```

Comment:
Does this belong in `tearDown`?

/cc @jmagman

Reply:
I'd have to move all their creations to setup if it was in tearDown

Reply:
more specifically, the message handler is the assertion mechanism in this test. It makes the test fail if I did dismiss the FlutterVC before disconnecting it. And I wouldn't create the FlutterVC in setup (because that's half of the test).

Reply:
Ahh I see, there's only one test in here anyway.  I was assuming there were multiple tests in this file.